app/views/_components/banner/template.njk

```diff
@@ -1,14 +1,4 @@
 <div class="app-banner{% if params.type == "success" %} app-banner--success{% elif params.type == "warning" %} app-banner--warning{% endif %}{{- " " + params.classes if params.classes}}" {%- for attribute, value in params.attributes -%} {{ attribute }}="{{ value }}"{% endfor %}>
-{% if params.icon != false %}
```

Comment:
I think the icons are valuable and that we should show them in order to provide a comparable experience to users who don't see colour so well.

Reply:
Have restored the ability to include icons with the banner component, but disabled to main parity with what’s shipped (not adding/changing features in this PR, just making things work!)